src/pages/RegisterChallengePage.vue

```diff
@@ -278,7 +295,7 @@ export default defineComponent({
                 unelevated
                 rounded
                 outline
-                @click="step = 3"
+                @click="onContinue"
```

Comment:
Shouldn't it be `step = 3`? Then the back button works as expected.

Reply:
That's right. My mistake on the button action.
I updated all of them using `onContinue` and `onBack` functions.